### ISSUE TITLE
value-list-max-empty-lines

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1713]
Limit the number of adjacent empty lines within value list.
Example:
With `0` valid:
```css
box-shadow:
        inset 0 2px 0 #dcffa6,
        0 2px 5px #000;
```
Invalid:
```css
box-shadow:
        inset 0 2px 0 #dcffa6,

        0 2px 5px #000;
```

**Comment by jeddy3** [comment]
SGTM.

**Comment by ntwb** [comment]
Nope, sounds good 👍

**Comment by davidtheclark** [comment]
No, none.

**Comment by jeddy3** [comment]
- Name: `value-list-max-empty-lines`
- Primary option: `int`
- Secondary options: none
- Message: `Expected no more than ${max} empty line(s)`
- Section: "Value list"
